lib/bundler/resolver.rb

```diff
@@ -250,28 +250,34 @@ def search_for(dependency)
         if vertex = @base_dg.vertex_named(dependency.name)
           locked_requirement = vertex.payload.requirement
         end
-        if results.any?
-          nested = []
-          results.each do |spec|
-            version, specs = nested.last
-            if version == spec.version
-              specs << spec
-            else
-              nested << [spec.version, [spec]]
+        spec_groups = begin
```

Comment:
yeah, I did that because without, then RubyMine indents the whole thing which makes Rubocop mad. So rather than figure out how to get RubyMine to make like Rubocop in this one case, I've taken to this. But it's sloppy, you're right.